app/utils/project.js

```diff
@@ -371,6 +371,27 @@ export default class ProjectUtil {
     ProjectUtil._sortAndAddFilter(ioFilter, filters);
     ProjectUtil._sortAndAddFilter(dependencyFilter, filters);
 
+    // Add attribute filter category last
+    const attributeFilter = { category: Constants.FilterCategory.ATTRIBUTE, values: [] };
+    const attributeFunc = (x) => {
+      if (!x.attributes) return null;
```

Comment:
I realize this isn't documented or in the linter (sorry), but my personal preference is for all conditional statements to use braces, even when they have a single-line.